documentation/samples/snippets/card.ts

```diff
@@ -1,59 +1,67 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import * as coda from "@codahq/packs-sdk";
 
 const pack = coda.newPack();
 
+const $1Schema = undefined;
+
 // BEGIN
 
 // A schema that defines the data shown in the card.
-const MyCardSchema = coda.makeObjectSchema({
+const $1$Thing$Schema = coda.makeObjectSchema({
   properties: {
-    property1: { type: coda.ValueType.String },
-    property2: { type: coda.ValueType.Number },
-    // Add more properties here.
+    $2$name$: { type: coda.ValueType.String },
+    $3$description$: { type: coda.ValueType.String },
+    $4$picture$: {
+      type: coda.ValueType.String,
+      codaType: coda.ValueHintType.ImageReference,
+    },
+    $5$link$: {
+      type: coda.ValueType.String,
+      codaType: coda.ValueHintType.Url,
+    },
+    // TODO: Add more properties.
   },
-  displayProperty: "<Property name>", // Display value shown in mention chip.
-  titleProperty: "<Property name>", // Title of the card.
+  displayProperty: "$2",
+  titleProperty: "$2",
+  snippetProperty: "$3",
+  imageProperty: "$4",
+  linkProperty: "$5",
   subtitleProperties: [
-    // Properties shown in the subtitle of the card.
-    "<Property name>",
-    { property: "<Property name>", label: "" }, // Show the value only.
-    // Add more subtitle properties here.
+    // TODO: List the properties to show under the title.
   ],
-  snippetProperty: "<Property name>", // Content shown in the card body
-  imageProperty: "<Property name>", // Image shown on the card.
-  linkProperty: "<Property name>", // Link opened when the card is clicked.
 });
 
 // A formula that accepts a URL and returns an object matching the schema above.
 pack.addFormula({
-  name: "<User-visible name of formula>",
-  description: "<Help text for the formula>",
+  name: "$1",
+  description: "${6:My description.}",
```

Comment:
How come this goes from 1 to 6?

Is it because the user will fill in the value for $1 on line 8 and it'll autopopulate it here?

Reply:
Yes, in this case `$1` is just referring to the value previously entered above.